package.json

```diff
@@ -40,6 +40,8 @@
     "easy-extender": "^2.3.1",
     "eazy-logger": "^2.1.2",
     "emitter-steward": "^0.0.1",
+    "engine.io": "automattic/engine.io",
```

Comment:
This seems dangerous for me because it always installs the latest commit. Installing specific working commit would be safer.

```
"engine.io": "automattic/engine.io#860af4dc0b47e55a623e7e56ce097ba4b610264b"
```